fix: make adapter plugin param type optional (#11246)
sveltejs/kit: .changeset/hungry-moles-turn.md

---
'@sveltejs/adapter-cloudflare-workers': patch
---

> fix: the adapter plugin parameter is now optional

Reviewer: ```suggestion
fix: declare the adapter plugin options as optional
```
